docs/en/operation/StressBench.md

```diff
@@ -361,17 +403,26 @@ Parameters:
 
 #### `RegisterWorkerBench` 
 
-For the `RegisterWorker` RPC and the `RequestRegisterLease` RPC.
+Simulates workload for the `RegisterWorker` RPC.
 
-This benchmark calls the RPC only once.
+This benchmark simulates concurrent worker registration using the unary RPC implementation (default 
+before Alluxio 2.7).
+You may use this command to simulate the pressure when the workers start at once at your scale.
 
-This benchmark measures the performance of the process of new worker registration. 
-The`RegisterWorker` RPC carries a list of blocks that is currently stored in the worker's storage. 
-This RPC can be expensive if the list is huge. 
-The `RequestRegisterLease` RPC is part of the mechanism that the master uses to 
-control the number of concurrently registering clients so that it does not exceed the master's 
+The`RegisterWorker` RPC carries a list of blocks that is currently stored in the worker's storage.
+This RPC can be expensive if the list is huge.
+The `RequestRegisterLease` RPC is part of the mechanism that the master uses to
+control the number of concurrently registering workers so that it does not exceed the master's
```

Comment:
The RPC is called `GetRegisterLeasePRequest` but that's too internal to the tech details. You can just remove these 2 lines, You are referring to the register lease doc below so I think it's fine.